scripts/build_chicoma_hackathon.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Make sure all the paths are correct
+
+rm -r build
+#rm -r install_magma_2.7.2
+rm -r install_hackathon
+
+MY_PATH=$(pwd)
+
+
+export CC=${CC:=cc}
+export FC=${FC:=ftn}
+export CXX=${CXX:=CC}
+#export CMAKE_Fortran_FLAGS=${CMAKE_Fortran_FLAGS:="-ef -DCRAY_SDK"}
```

Comment:
A lot of commented lines in this file too...

Reply:
This has been removed